File: src/IntegraApp.Application/Services/Empleado/Commands/CreateEmpleadoCommand/CreateEmpleadoModel.cs
using Microsoft.AspNetCore.Http;

namespace IntegraApp.Application.Services.Empleado.Commands.CreateEmpleadoCommand
{
    public class CreateEmpleadoModel
    {
        public string? Apellido { get; set; }
        public string? Nombre { get; set; }
        public string? Telefono { get; set; }
        public string? Correo { get; set; }
        public string? FotoPath { get; set; }
        public DateTime FechaContratacion { get; set; }
    }
}

File: IntegraAppFront/src/app/models/Empleado/create-empleado.model.ts
export class CreateEmpleado {
  apellido: string | null;
  nombre: string | null;
  telefono: string | null;
  correo: string | null;
  fechaContratacion: Date;
  archivo: File | null;

  constructor(
    apellido: string | null,
    nombre: string | null,
    telefono: string | null,
    correo: string | null,
    fechaContratacion: Date,
    archivo: File | null
  ) {
    this.apellido = apellido;
    this.nombre = nombre;
    this.telefono = telefono;
    this.correo = correo;
    this.fechaContratacion = fechaContratacion;
    this.archivo = archivo;
  }
}
